apps/testingaccessibility/src/pages/api/stripe/webhook.ts

```diff
@@ -50,6 +53,14 @@ const stripeWebhookHandler = async (
 
         await postSaleToSlack(purchaseInfo, purchase)
 
+        res.status(200).send(`This works!`)
+      } else if (event.type === 'charge.refunded') {
+        const chargeId = event.data.object.id
+        await updatePurchaseStatusForCharge(chargeId, PurchaseStatus.Refunded)
+        res.status(200).send(`This works!`)
+      } else if (event.type === 'charge.dispute.created') {
+        const chargeId = event.data.object.id
+        await updatePurchaseStatusForCharge(chargeId, PurchaseStatus.Disputed)
```

Comment:
when charges are disputed or refunded we want to mark them so so we can limit access as required